inc/spbc-settings.php

```diff
@@ -188,7 +188,7 @@ function spbc_settings__register() {
 				'icon' => 'spbc-icon-sliders',
 				'class_prefix' => 'spbc',
 				'ajax' => true,
-				'js_after'  => 'settings_tab--settings_general.min.js',
+				'js_after'  => 'settings_tab--settings_general.js',
```

Comment:
I think this was made for debug purposes. Please, return the using of minimized script.

Reply:
it was fixed earlier